### PR TITLE
add append mode to LABEL

### PR DESCRIPTION
**Pull request by larkob**
I needed a simple way to append text to a label rather than to replace it, so I added another mode.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3321/pr-test (or any other room on that server)

**Review by ArnoldSmith86 — APPROVED** [review]
Didn't test but the code looks fine.

File: client/js/widgets/button.js [hunk 1]
```diff
@@ -385,7 +385,7 @@ export class Button extends Widget {
 
       if(a.func == 'LABEL') {
         setDefaults(a, { value: 0, mode: 'set', collection: 'DEFAULT' });
-        if([ 'set', 'dec', 'inc' ].indexOf(a.mode) == -1)
+        if([ 'set', 'dec', 'inc','append' ].indexOf(a.mode) == -1)
           problems.push(`Warning: Mode ${a.mode} will be interpreted as add.`);
```
Comment (rogerl50) [inline client/js/widgets/button.js:389]: The default is 'set' in the call to `problems`. Looks like this line was copy-pasted from SELECT?

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3321/pr-test (or any other room on that server)

**Pull request merged**